# Params for TIP planner
debug: true
# use_memory: true
safe_distance: 1.5   # Smallest distance before obstacle is no longer viable segment
sensor_distance: 3 # Primitives evaluated up to this distance
mem_distance: 0.5
buffer: 0.4        # 0.5 width of quad for collision checking

# (x,y,z) goal
goal_x: 0.0
goal_y: -20.0
goal_z: 1

# Flight params
max_speed: 1.5

jerk: 15  
accel: 5.0
accel_stop: 5.0

# Trajectory evaluation rate
plan_eval: 0.01

# Number of samples from k-d tree to find obstacle 
K: 10

# Sensor horizontal FOV (deg)
h_fov: 50

# Number of horizontal FOV discretization
h_samples: 10

# Sensor vertical FOV (deg)
v_fov: 40

# Number of vertical FOV discretization
v_samples: 8

# Max yaw rate (rad/s)
r_max: 1

# Maximum tolerable jump in state estimate before replanning
jump_thresh: 0.5

# Min and max allowable z values 
z_min: 0.2
z_max: 1.5

# Goal buffer
goal_radius: 0.5
    
